crates/orderbook/src/database/orders.rs

```diff
@@ -76,13 +75,44 @@ pub trait OrderStoring: Send + Sync {
         limit: Option<u64>,
     ) -> Result<Vec<Order>>;
     async fn latest_order_event(&self, order_uid: &OrderUid) -> Result<Option<OrderEvent>>;
+    async fn single_order_with_quote(&self, uid: &OrderUid) -> Result<Option<OrderWithQuote>>;
 }
 
 pub struct SolvableOrders {
     pub orders: Vec<Order>,
     pub latest_settlement_block: u64,
 }
 
+pub struct OrderWithQuote {
```

Comment:
Created a new object which contains an order with the quote information (the quote information corresponding to the one stored in `order_quotes` table, which is the only info we need, not the full quote).